Dockerfile

```diff
@@ -1,36 +1,57 @@
-# specify the node base image with your desired version node:<version>
-FROM node:12.16.3-slim
+# Build Stage
+FROM node:12.16.3-buster-slim AS umbrel-manager-builder
 
-# Install Tools
+# Install tools
 RUN apt-get update --no-install-recommends \
-    && apt-get install -y --no-install-recommends build-essential g++ \
-    && apt-get install -y --no-install-recommends git \
-    && apt-get install -y --no-install-recommends libltdl7 \
-    && apt-get install -y --no-install-recommends python \
-    && apt-get install -y --no-install-recommends rsync \
-    && apt-get install -y --no-install-recommends vim \
+    && apt-get install -y --no-install-recommends build-essential \
+    && apt-get install -y --no-install-recommends g++ \
+    && apt-get install -y --no-install-recommends make \
+    && apt-get install -y --no-install-recommends libssl-dev \
+    && apt-get install -y --no-install-recommends libffi-dev \
+    && apt-get install -y --no-install-recommends python3-dev \
+    && apt-get install -y --no-install-recommends python3-setuptools \
+    && apt-get install -y --no-install-recommends python3-wheel \
     && apt-get install -y --no-install-recommends python3 \
-    && apt-get install -y --no-install-recommends libssl-dev libffi-dev python3-dev python3-setuptools python3-wheel python3-pip \
+    && apt-get install -y --no-install-recommends python3-pip \
     && pip3 install -IU docker-compose \
     && ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose \
```

Comment:
What's this symlink for? I think we can remove it.

Reply:
Good catch, it's redundant